phpBB/posting.php

```diff
@@ -14,6 +14,9 @@
 /**
 * @ignore
 */
+ini_set('display_errors', 1);
```

Comment:
Remove this. You can enable the development enviroment in your config.php so all notices are shown.